### PR TITLE
Changes to the custom item.

### PR DESCRIPTION
**Pull request by EllyBrown**
Changed sprite and some of text, representing it.

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_fluffitems_vr.dm [hunk 1]
```diff
@@ -1006,8 +1006,8 @@
 	ckeywhitelist = list("zodiacshadow")
 	character_name = list("Nehi Maximus")
 
-/datum/gear/fluff/star_sweater
-	path = /obj/item/clothing/accessory/sweater/fluff/star
-	display_name = "Star Sweater"
+/datum/gear/fluff/star_hoodie
+	path = /obj/item/clothing/suit/storage/toggle/hoodie
```
Comment (Novacat) [inline code/modules/client/preference_setup/loadout/loadout_fluffitems_vr.dm]: add /star_hoodie to the end of this

File: code/modules/vore/fluffstuff/custom_clothes_vr.dm [hunk 2]
```diff
@@ -1900,15 +1900,15 @@ Departamental Swimsuits, for general use
 	return FALSE	//Indigestible
 
 //Bacon12366:Elly Brown
-/obj/item/clothing/accessory/sweater/fluff/star
-	name = "Star Sweater"
-	desc = "It's a black long sweater with a big blue star at the chest area. It was made to show person's left shoulder."
+/obj/item/clothing/suit/storage/toggle/hoodie
```
Comment (Novacat) [inline code/modules/vore/fluffstuff/custom_clothes_vr.dm]: Same here. /star_hoodie

**Comment by Heroman3003** [comment]
There are already hoodies in the game. Might wanna make it a /hoodie/fluff/star_hoodie subtype or something like that.

File: code/modules/vore/fluffstuff/custom_clothes_vr.dm [hunk 3]
```diff
@@ -1900,20 +1900,19 @@ Departamental Swimsuits, for general use
...
+	icon_state = "star_hoodie"
 
 
 	icon_override = 'icons/vore/custom_clothes_vr.dmi'
-	item_state = "star_sweater"
+	item_state = "star_hoodie"
 
 	slot_flags = SLOT_OCLOTHING | SLOT_TIE
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
 	w_class = ITEMSIZE_NORMAL
```
Comment (Novacat) [inline code/modules/vore/fluffstuff/custom_clothes_vr.dm]: You can remove these lines since they are no longer necessary.

**Pull request merged**